src/main/java/sirius/web/data/XLSProcessor.java

```diff
@@ -63,6 +93,7 @@ protected Workbook openWorkbook() throws IOException {
     }
 
     protected void importSheet(RowProcessor rowProcessor, Predicate<Exception> errorHandler, Sheet sheet) {
+        currentSheet = sheet.getSheetName();
```

Comment:
I'd not make this class stateful. Couldn't we rather subclass RowProcessor into "SheetBasedRowProcessor" which accepts an additional parameter - we could then check which class is present below and invoke the proper function...

Reply:
Or we could invoke a beginSheet(Sheet) / endSheert(Sheet) if present....

Reply:
The SheetBasedRowProcessor approach makes this a lot cleaner. Thanks for the suggestion 👍